application/schemas/artifact_schema.py

```diff
@@ -1,11 +1,12 @@
-""" Defines schema for database Artifact objects """
+""" Defines schema for database artifact objects """
```

Comment:
I mean the class is literally named Artifact. I think keeping it with a capital makes a lot of sense.

Reply:
just consistency with the other files arounf it. They are usually non-capital artifacts in the docstring (see controller and model).
I really don't feel strongly about this so I'm fine with "Artifacts".